File: src/main/java/scala/Student.java
package scala;

public class Student {

    private String name;

    private Integer age;

    private static String school = "atguigu";

    public Student(String name, Integer age) {
        this.name = name;
        this.age = age;
    }

    public void printInfo() {
        System.out.println(this.name + " " + this.age + " " + Student.school);
    }
}

File: src/main/scala/chapter01/Student.scala
package chapter01

class Student(name: String, age: Int) {

  def printInf(): Unit = {
    println(name + " " + age + " " + Student.school);
  }
}

// 引入伴生对象
object Student {

  val school: String = "atguigu"
}
